docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst

```diff
@@ -35,12 +35,72 @@ Child Growth Failure Risk Correlations
 Risk Exposures Overview
 ------------------------
 
+Child anthropometry is measured through several different correlated metrics, generally related to length/height, weight, and age. Relevant metrics include:
+
+.. list-table:: Child Anthropometry Metrics
+   :header-rows: 1
+
+   * - Metric
+     - Typical threshold value
+   * - Gestational age
+     - Preterm birth: <37 weeks
+   * - Birthweight
+     - Low birthweight: <2500 grams
+   * - Length/height for age (LAZ/HAZ)
+     - Stunted: 2+ standard deviations below mean
+   * - Weight for length/height (WLZ/WHZ)
+     - Wasted: 2+ standard deviations below mean
+   * - Weight for age (WAZ)
+     - Underweight: 2+ standard deviations below mean
+
+Notably, these metrics tend to be highly correlated, with the magnitude of correlation varying with age. The metrics are related both through common causes as well as through suspected direct causal influences in some cases. The below diagram represents a causal framework for child growth metrics at birth as they relate to child growth metrics throughout the pre- and post-natal periods, as presented by [Na-et-al-2020]_.
+
+.. image:: dag_framework.png
+
+Notably, it is critical to distinguish between the correlation between these metrics attributable to shared common causes between them from correlation between these metrics attributable to direct causal relationships. However, the influence of confounding factors in the estimation of the causal relationship between these factors must be carefully considered due to the high correlation between them and the observational nature of child antropometry studies. Additionally, child age should be considered as an important effect modifier in the associations between each of these metrics, as discussed by [Alderman-and-Headey-2018]_.
+
 Correlation
 ++++++++++++
 
+Spearman correlation coefficients between birthweight and LAZ, WAZ, and WLZ among the first two years of age were estimated from the MAL-ED study [MAL-ED-investigators-2014]_, available in :download:`this spreadsheet <child_growth_birthweight_correlations.xlsx>` and presented in the figures below.
+
+.. image:: bw_laz_correlation_plot.png
+
+.. image:: bw_waz_correlation_plot.png
+
+.. image:: bw_wlz_correlation_plot.png
+
+.. note::
+
+   These correlation coefficients were estimated by Chris for previous work on the :ref:`risk correlation proposal <2017_risk_models>` for the :ref:`Balanced Energy Protein simulation <2017_concept_model_vivarium_gates_bep>`.
+
```

Comment:
Perhaps reference his full name and team for future researchers.

Reply:
I second Nicole's comment!